tests/conftest.py

```diff
@@ -436,3 +444,7 @@ def tag_test_report(request, pytestconfig, testbed, duthost, record_testsuite_pr
         record_testsuite_property("os_version", duthost.os_version)
 
         __report_metadata_added = True
+
+@pytest.fixture(scope="module")
+def skip_cleanup(request): 
```

Comment:
This seems like it could be handy to have in general, but for now I would also move it to a `vxlan` specific conftest.

Reply:
If I move this to a `vxlan` specific conftest could we potentially run into a naming conflict if another test also implements a `--skip_cleanup` option?